bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/container.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { withTracker } from 'meteor/react-meteor-data';
+import Meetings from '/imports/api/meetings';
+import UserParticipants from './component';
+
+const UserParticipantsContainer = ({ ...props }) => <UserParticipants {...props} />;
+
+export default withTracker(({ getUsersId }) => ({
+  meeting: Meetings.findOne({}),
```

Comment:
I tried following why we need the meeting passed (not just the id) and it seems it's not needed in UserParticipants component. It's passed to UserOptionsContainer but there it's also obtained from meetingId anew. Seems to me we don't need to pass the whole meeting. Please comment